buildspec.yml

```diff
@@ -0,0 +1,56 @@
+version: 0.2
+
+phases:
+  install:
+    runtime-versions:
+      golang: 1.13
+  pre_build:
+    commands:
+      - echo "cd into $CODEBUILD_SRC_DIR"
+      - cd $CODEBUILD_SRC_DIR
+      - export GOMOD=$CODEBUILD_SRC_DIR/go.mod:GOMOD
+      - export GOPATH="/go"
+  build:
+    commands:
+      - echo "Compilation context:"
+      - echo "CODEBUILD_SOURCE_VERSION=$CODEBUILD_SOURCE_VERSION"
+      - make gen-mocks
```

Comment:
Do we need to run make tools??

Reply:
Doesn't gen-mocks run tools as a prerequisite? Or is it better to explicitly run it?